### PR TITLE
WinUI-3215 : Update the features in rating and shadow control

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Prepared and Updated the Rating control orientation support and Shadow control enable shadow effect support in WinUI in UG documentation.

### CODE REVIEW

File: winui/Rating/Customization.md [hunk 1]
```diff
@@ -108,6 +108,30 @@ rating.ItemSize = 50;
 
 ![Size customization in WinUI Rating control](Rating_images/winui_rating_itemsize.png)
 
+## Orientation 
+
+The Rating control provides support to displayed horizontally or vertically by setting the `Orientation` property.
+
```
Comment (Manivannan-E) [inline winui/Rating/Customization.md:114]: @Priyanka-Vijayasankar The Rating control provides support to display the rating items either horizontally or vertically by setting the `Orientation` property. The default value is horizontal.

File: winui/Shadow/Customization.md [hunk 2]
```diff
@@ -128,3 +128,61 @@ shadow.Content = button;
 {% endtabs %}
 
 ![Positioning in WinUI Shadow control](Shadow_images/winui_shadow_offset.png)
+
+## Enable or Disable the shadow
+
+The enable or disable the shadow can be customized using the `EnableShadow` property. The default value of the `EnableShadow` property is true.
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:134]: @Priyanka-Vijayasankar The Shadow control provides support to enable or disable the shadow effect by using the `EnableShadow` property. The default value of the `EnableShadow` property is true.

File: winui/Rating/Customization.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:114]: Changed the content as mentioned above.

File: winui/Shadow/Customization.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md]: Changed the content as mentioned above.

File: winui/Shadow/Customization.md [hunk 3]
```diff
@@ -128,3 +128,35 @@ shadow.Content = button;
 {% endtabs %}
 
 ![Positioning in WinUI Shadow control](Shadow_images/winui_shadow_offset.png)
+
+## Enable or Disable the shadow
+
+The Shadow control provides support to enable or disable the shadow effect by using the `EnableShadow` property. The default value of the `EnableShadow` property is true.
+
+The below code snippet show how to display the shadow to be false:
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:136]: @Priyanka-Vijayasankar The below code snippet shows how to disable the shadow effect:

File: winui/Shadow/Customization.md [hunk 4]
```diff
@@ -128,3 +128,35 @@ shadow.Content = button;
...
+Button button = new Button(); 
+button.Height = 50;
+button.Width = 100;
+button.Content = "Button";
+shadow.Content = button;
+
+{% endhighlight %}
+{% endtabs %}
+
+![EnableShadow false state WinUI Shadow control](Shadow_images/winui_shadow_enableshadow_false.png)
```
Comment (Manivannan-E) [inline winui/Shadow/Customization.md:162]: @Priyanka-Vijayasankar [EnableShadow in WinUI Shadow control]

File: winui/Shadow/Customization.md (hunk 3 above)
Comment (Priyanka-Vijayasankar) [reply winui/Shadow/Customization.md]: Changed the content as mentioned above.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**